console/src/config.rs

```diff
@@ -176,3 +191,13 @@ fn parse_true_color(s: &str) -> bool {
     let s = s.trim();
     s.eq_ignore_ascii_case("truecolor") || s.eq_ignore_ascii_case("24bit")
 }
+
+fn parse_retain_for(s: &str) -> Result<RetainFor, String> {
+    match s {
+        "none" => Ok(RetainFor(None)),
+        _ => match s.parse::<humantime::Duration>() {
+            Ok(duration) => Ok(RetainFor(Some(duration.into()))),
+            Err(error) => Err(error.to_string()),
```

Comment:
nit: i would probably change this to just return the `humantime` error:
```suggestion
fn parse_retain_for(s: &str) -> Result<RetainFor, humantime::Error> {
    match s {
        "none" => Ok(RetainFor(None)),
        _ => s
            .parse::<humantime::Duration>()
            .map(|duration| RetainFor(Some(duration.into()))),
```
or something?